### PR TITLE
Anvil loading delay settings

### PR DESCRIPTION
**Pull request by hanars**
Re-enables the loading warnings we had last year, and changes the mechanism to be a single env variable to turn on/off the message rather than requiring a full code deployment every year to turn this on. Text is based on what we did last year: https://github.com/broadinstitute/seqr/commit/518a1acb0eda7eaf24f50ddc7ab4577dcf76970d

### CODE REVIEW

File: seqr/views/react_app.py [hunk 1]
```diff
@@ -43,11 +44,15 @@ def render_app_html(request, additional_json=None, include_user=True, status=200
         ui_version = 'local'
         html = html.replace('</head>', '<script defer="defer" src="/app.js"></script></head>')
 
+    should_show_loading_delay = bool(ANVIL_LOADING_DELAY_EMAIL_DATE)
+    if should_show_loading_delay:
+        should_show_loading_delay = datetime.strptime(ANVIL_LOADING_DELAY_EMAIL_DATE, '%Y-%m-%d') < datetime.now()
```
Comment (ShifaSZ) [inline seqr/views/react_app.py:49]: How about moving the time checking to the front end?

File: seqr/views/apis/anvil_workspace_api.py [hunk 2]
```diff
@@ -33,14 +33,13 @@
 from seqr.utils.middleware import ErrorsWarningsException
 from seqr.views.utils.permissions_utils import is_anvil_authenticated, check_workspace_perm, login_and_policies_required
 from settings import BASE_URL, GOOGLE_LOGIN_REQUIRED_URL, POLICY_REQUIRED_URL, API_POLICY_REQUIRED_URL,\
-    SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, AIRFLOW_API_AUDIENCE, AIRFLOW_WEBSERVER_URL, SEQR_SLACK_LOADING_NOTIFICATION_CHANNEL
+    SEQR_SLACK_ANVIL_DATA_LOADING_CHANNEL, AIRFLOW_API_AUDIENCE, AIRFLOW_WEBSERVER_URL, ANVIL_LOADING_DELAY_EMAIL_DATE, \
```
Comment (ShifaSZ) [inline seqr/views/apis/anvil_workspace_api.py:36]: If you don't think it is too long, it is less confusing by changing the constant name to `ANVIL_LOADING_DELAY_EMAIL_START_DATE`.

**Review by ShifaSZ — CHANGES_REQUESTED** [review]
Two non-critical comments.

File: seqr/views/react_app.py (hunk 1 above)
Comment (hanars) [reply seqr/views/react_app.py:49]: I'm not in love with javascript's date processing and I'd rather leave this sort of business logic server side

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**